examples/multiphysics/rising_bubble_VOF/rising_bubble.py

```diff
@@ -0,0 +1,118 @@
+#############################################################################
+"""
+Postprocessing code for rising_bubble_VOF example
+
+"""
+#############################################################################
+
+'''Importing Libraries'''
+from math import pi
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib.tri as tri
+from matplotlib import cm
+import pyvista as pv
+from scipy.signal import savgol_filter
+
+import glob
+
+import os
+import sys
+from pyvista.plotting.renderer import CameraPosition
+
+from scipy.linalg.special_matrices import dft
+#############################################################################
+
+#############################################################################
+'''Functions'''
+
+#############################################################################
+
+#Take case path as argument and store at currentPath
+currentPath = sys.argv[1]
+
+#Define list of VTK files and time list:
+list_vtu = os.listdir(currentPath)
+list_vtu = [x for x in list_vtu if "pvd" not in x ]
+list_vtu = [x for x in list_vtu if "pvtu" not in x ]
+
+#Create a list of time_steps
+data = pd.read_csv("Output/rising_bubble_VOF.pvd",sep='"',header=5, usecols=[1]) 
+data.columns = ["a"] 
+time_list = data['a']
+print(data['a'])
+
+#Set phase_limit to search for maximum x
+phase_limit = 0.5
+
+#Sort list
+time_list, list_vtu = (list(t) for t in zip(*sorted(zip(time_list, list_vtu))))
+
+#Create lists to fill with average y, dy, dt and bubble rise velocity
+y_list = []
+y_diff_list = []
+t_diff_list = []
+bubble_rise_vel = []
+
+#Read vtu data
+for i in range(0, len(list_vtu)):
+    #Read DF from VTK files
+    exec(f'df_{i} = pv.read(f\'{currentPath}/{list_vtu[i]}\')')
```

Comment:
I don't understand why you can't put the filename+path in a string variable instead of calling an external exec?